### PR TITLE
Added rowHeight property for ListView.

### PR DESCRIPTION
**Pull request by nsndeck**

### CODE REVIEW

**Comment by enchev** [comment]
:+1:

File: ui/list-view/list-view.ios.ts [hunk 1]
```diff
@@ -212,6 +241,19 @@ export class ListView extends common.ListView {
         this._heights[index] = value;
     }
 
+    public _onRowHeightPropertyChanged(data: dependencyObservable.PropertyChangeData) {
+        if (data.newValue < 0) {
```
Comment (hshristov) [inline ui/list-view/list-view.ios.ts]: if newValue < 0 we should set height to:
tableView.rowHeight = UITableViewAutomaticDimension;
and estimatedRowHeight to 44 (default rowHeight)
We could also change our ESTIMATED_HEIGHT to 44.

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by ErjanGavalji** [comment]
:+1:  after applying @hshristov 's note

**Comment by nsndeck** [comment]
run ci

**Pull request merged**